### DISCUSSION TITLE
How to change the font size of the bottom tab labels?

### DISCUSSION THREAD

**Discussion by fredrikburmester** [callstack/react-native-bottom-tabs#171] [Q&A]
After upgrading from 0.4.0 to 0.7.1 the font size is quite large. How can i customize the font size? 

Using Expo.

https://github.com/fredrikburmester/streamyfin/blob/master/app/(auth)/(tabs)/_layout.tsx

**Comment by okwasniewski** [comment]
Hey, there are new font related options: https://okwasniewski.github.io/react-native-bottom-tabs/docs/guides/usage-with-react-navigation.html#tablabelstyle

If the font changed from the default size it's probably a regression I will check it
